lib/teleterm/gateway/gateway.go

```diff
@@ -119,9 +119,21 @@ func (g *Gateway) Open() {
 	}()
 }
 
+// LocalPortInt returns the port of a gateway as an integer rather than a string.
+func (g *Gateway) LocalPortInt() int {
+	// Ignoring the error here as Teleterm doesn't allow the user to pick the value for the port, so
+	// it'll always be a random integer value, not a service name that needs actual lookup.
+	// For more details, see https://stackoverflow.com/questions/47992477/why-is-port-a-string-and-not-an-integer
+	port, _ := net.LookupPort("", g.LocalPort)
```

Comment:
nit: Why do we need `net.LookupPort` here? This function can resolve ports from `/etc/services` for example `net.LookupPort("tcp", "http")` return `80`.
In this case I think that https://pkg.go.dev/strconv#Atoi is more appropriate.

Reply:
In general there's a couple of places where Teleterm's Go code stores the port as a string rather than an integer. I don't know what the actual rationale for that is, but I suspect it's just convenient because it uses the net package to get the port string in the first place. So I thought that in order to be consistent with the net package usage, I'll just use `LookupPort` here for that conversion.

But yeah, I don't think we plan to support service names here so I'll just use `Atoi` here. Changing ports to ints in Teleterm would be a lot of work at this point.